scipy/spatial/transform/_rigid_transform.py

```diff
@@ -1707,6 +1772,19 @@ def __reduce__(self) -> tuple[Callable, tuple]:
             matrix = matrix[0, ...]
         return (self.__class__.from_matrix, (matrix,))
 
+    @xp_capabilities(jax_jit=False)
```

Comment:
I assume it is an artefact of an iterator object not being able to get out of the JIT? This function looks JIT-compatible to me; it's just wrapping it individually in the jit that is not possible.
This feels like a use case where `xp_capabilities` may not be a good fit.

Reply:
Upstream fix: https://github.com/data-apis/array-api-extra/pull/418